java/src/main/native/src/TableJni.cpp

```diff
@@ -1496,6 +1497,43 @@ JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_readParquet(JNIEnv *env,
   CATCH_STD(env, NULL);
 }
 
+JNIEXPORT jlongArray JNICALL Java_ai_rapids_cudf_Table_readAvro(
+  JNIEnv *env, jclass, jobjectArray filter_col_names, jstring inputfilepath, jlong buffer, jlong buffer_length, jint unit) {
```

Comment:
This line has more than 100 chars.
<= 100 is preferred

Reply:
Done